versus-fe/src/App.js

```diff
@@ -25,23 +25,20 @@ class App extends Component {
 
   render() {
     if (this.props.error) return <ErrorPage />;
-    else if (this.props.loading && !this.props.user.user_id)
-      return <Loading />;
+    else if (this.props.loading && !this.props.user.user_id) return <Loading />;
     else
-      return(
+      return (
         <div className="App">
-
           <Route path="/" component={Drawer} />
           <Route path="/login" component={LoginPage} />
           <Route path="/signUp" component={SignUp} />
-          <Route path="/matches" component={Matches} />
+          <Route path="/myMatches" component={Matches} />
```

Comment:
I preferred the `matches` version. Is like requesting the `/profile` (`/myProfile`?) or `/opponents` (`myOpponents`)